packages/pyroscope-flamegraph/src/FlameGraph/FlameGraphComponent/index.tsx

```diff
@@ -265,11 +265,20 @@ export default function FlameGraphComponent(props: FlamegraphProps) {
         );
       };
 
+      const ResetItem = () => {
+        const isResetDisabled =
+          !(selectedItem.isJust && selectedItem.value) && !dirty;
```

Comment:
why do we need to check `selectedItem.value` exists? if it's not a `Nothing` then it should have a value

Reply:
correct, i will fix it